apps/web-app/src/app/group/page.tsx

```diff
@@ -23,6 +23,8 @@ export default function GroupsPage() {
         }
     }, [_users, setLog])
 
+    const users = [..._users].reverse();
```

Comment:
`useMemo` should reduce the number of assignments here I guess. Same for feedback.

Reply:
correct, changes done